e2e/test/scenarios/models/reproductions.cy.spec.js

```diff
@@ -1335,6 +1335,92 @@ describe("issue 53556 - nested question based on native model with remapped valu
   });
 });
 
+describe("issue 53604 - nested native question with multiple breakouts on same column", () => {
+  const questionDetails = {
+    name: "53604 base",
+    type: "question",
+    native: {
+      query: "select ID, CREATED_AT from ORDERS",
+      "template-tags": {},
+    },
+  };
+
+  function createNestedQuestion({
+    turnIntoModel: shouldTurnIntoModel = false,
+  } = {}) {
+    H.createNativeQuestion(questionDetails).then(({ body: { id } }) => {
+      if (shouldTurnIntoModel) {
+        H.visitQuestion(id);
+        turnIntoModel();
+      }
+      H.createQuestion(
+        {
+          type: "question",
+          name: "53604",
+          query: {
+            "source-table": `card__${id}`,
+            aggregation: [["count"]],
+            breakout: [
+              [
+                "field",
+                "CREATED_AT",
+                { "temporal-unit": "month", "base-type": "type/DateTime" },
+              ],
+              [
+                "field",
+                "CREATED_AT",
+                { "temporal-unit": "year", "base-type": "type/DateTime" },
+              ],
+            ],
+          },
+          display: "line",
+        },
+        {
+          wrapId: true,
+          idAlias: "nestedQuestionId",
+        },
+      );
+    });
+  }
+
+  function testUnderlyingRecordsDrillThrough() {
```

Comment:
```suggestion
  function testUnderlyingRecordsDrillThru() {
```
is the form usually used.

Reply:
fixed